ROLES.md

```diff
@@ -242,7 +274,89 @@ The following apply to the area / component for which one would be an owner.
 
   - Fixing bugs.
 
-  - Mentoring and guiding approvers, members, and contributors.
+  - Identifying needed enhancements / areas for improvement / etc.
+
+  - Execute pay-down of technical debt.
+
+- Mentoring and guiding approvers, members, and new contributors.
+
+## Working Group Execution Lead
+
+Working group execution leads, or just ‘execution leads’, are responsible for
+the overall health and execution of the working group itself. Execution leads
+work with tech leads to ensure that the working group is making progress toward
+its goals, is aligned with the project roadmap, etc. The execution lead may also
+be the tech lead in a smaller working group, but they are distinct roles.
```

Comment:
🤔 Smaller working group?  We don't have a hierarchy, so I find this confusing.

Reply:
example: Serving is bigger than operations

Reply:
I see, I read it as a `sub-group`, perhaps `s/a smaller/another/`?

Reply:
Here we meant that in workgroups with less people involved, the same individual will play both roles. Larger groups, potentially like Serving, can benefit from the specialization of roles.